policies/policies.go

```diff
@@ -35,23 +35,33 @@ import (
 )
 
 func run(ctx context.Context) {
+	var resp *agentendpointpb.EffectiveGuestPolicy
+
 	client, err := agentendpoint.NewClient(ctx)
 	if err != nil {
 		logger.Errorf("agentendpoint.NewClient Error: %v", err)
-		return
+	} else {
+		defer client.Close()
+		resp, err = client.LookupEffectiveGuestPolicies(ctx)
+		if err != nil {
+			logger.Errorf("Error running GuestPolicies: %v", err)
+			resp = &agentendpointpb.EffectiveGuestPolicy{}
```

Comment:
Just have mergeConfigs check for a nil EffectiveGuestPolicy so we don't need to do this here